### ISSUE TITLE
pre-rendering mathjax content

### ISSUE THREAD

**Issue by dverma4u** [calvinkei/react-native-mathjax#14]
is there any way to pre-render the data with mathjax, so that user won't see the loading of mathjax content?

**Comment by calvinkei** [comment]
please update to 2.0.0 to check if the issue persist

**Comment by dverma4u** [comment]
@calvinkei yes the new version is faster. But my issue is to load all the content before displaying them to the user.

current scenario is that if user's internet connecting goes away he may see old latex data. or if the user's internet connection is slow he may have to see that conversion of latex to mathjax equation. 

can't we pre-fetch all the equations before displaying them to the user?

**Comment by calvinkei** [comment]
updated at 44850fc05b025b4532c5b078d32b407bad0b0bba, please check if it fits your need and I will publish it to 2.0.1 once you confirmed it is solved

**Comment by dverma4u** [comment]
i think this hide content until loading complete was available in previous version also. 
yes my pain point is on internet connection.
 i'll try to put mathjax scripts in ./assets folder . Let's see if it helps or not.

**Comment by truongluuxuan** [comment]
@dverma4u where ./assets in ios?

**Comment by shivam0831** [comment]
any solution?
